src/renderers/webgl/WebGLUniformsGroups.js

```diff
@@ -126,7 +126,8 @@ function WebGLUniformsGroups( gl, info, capabilities, state ) {
 						uniform.__data[ 0 ] = value.elements[ 0 ];
 						uniform.__data[ 1 ] = value.elements[ 1 ];
 						uniform.__data[ 2 ] = value.elements[ 2 ];
-						uniform.__data[ 3 ] = value.elements[ 0 ];
+						// Conversion to 3x4 matrix
+						uniform.__data[ 3 ] = 0;
```

Comment:
Need `uniform.__data[ 7 ]` and `uniform.__data[ 11 ]` also the `0` assignment?

Reply:
Ah yes indeed, I believe that otherwise it would add an unwanted translation to the converted matrice. Will update!